platform/ui/src/components/studyBrowser/Thumbnail.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDrag } from 'react-dnd';
+import ImageThumbnail from './ImageThumbnail';
+import classNames from 'classnames';
+
+import './Thumbnail.styl';
+
+function ThumbnailFooter({ seriesDescription, seriesNumber, instanceNumber, numImageFrames }) {
+  const infoOnly = !seriesDescription;
+
+  return (
+    <div className={classNames('series-details', { 'info-only': infoOnly })}>
+      <div className="series-description">{seriesDescription}</div>
+      <div className="series-information">
```

Comment:
`series-information` looks to contain 3 values. Can use flexbox and `justify-content` to align these properly?

Also, if all three do not have values, can we conditionally render `series-information`? (This is common w/ `SR`)

Reply:
ok